src/pages/home/report/ReportActionsList.js

```diff
@@ -366,6 +366,19 @@ function ReportActionsList({
         [onLayout],
     );
 
+    const listHeaderComponent = useCallback(() => {
+        if (firstRenderRef.current) {
+            firstRenderRef.current = false;
+            return null;
+        }
```

Comment:
i just returned previous comment to this code
``` 
 // Skip this hook on the first render, as we are not sure if more actions are going to be loaded
 // Therefore showing the skeleton on footer might be misleading

Reply:
I got that but since we are using the same ref in two functions. One of the functions will set that ref to false and the other will read it as false (instead of as true) and we won't be skipping the first render.

Reply:
I got it. 
I just added a separate prop for this, thanks